pkg/kubelet/kubelet.go

```diff
@@ -2770,6 +2770,7 @@ func (kl *Kubelet) canResizePod(pod *v1.Pod) (bool, *v1.Pod, v1.PodResizeStatus)
 
 	node, err := kl.getNodeAnyWay()
 	if err != nil {
+		kl.recorder.Eventf(pod, v1.EventTypeWarning, events.ResizeFailed, "Resize pod %s failed: %v", klog.KObj(pod), err)
```

Comment:
This has to do with getting the nodeinfo and the error is not strictly related to pod resizing